src/config.rs

```diff
@@ -2370,6 +2370,19 @@ impl TiKvConfig {
         if kv_db_path == self.raft_store.raftdb_path {
             return Err("raft_store.raftdb_path can not same with storage.data_dir/db".into());
         }
+        let kv_db_wal_path = if self.rocksdb.wal_dir.is_empty() {
+            config::canonicalize_path(&kv_db_path)?
+        } else {
+            config::canonicalize_path(&self.rocksdb.wal_dir)?
+        };
+        let raft_db_wal_path = if self.raftdb.wal_dir.is_empty() {
+            config::canonicalize_path(&self.raft_store.raftdb_path)?
+        } else {
+            config::canonicalize_path(&self.raftdb.wal_dir)?
+        };
+        if kv_db_wal_path == raft_db_wal_path {
```

Comment:
sometimes relative addr may be used and it will be sikpped in `canonicalize_sub_path()`. For example, `a!=xx/../a`

Reply:
Yep, I reviewed this and found you're right. I'll restore the code here. @tabokie @hunterlxt

Reply:
seem that this issue existed before the PR 😅

Reply:
I'm sorry could you please show the issue below? I failed to found the issue you mentioned either associated with the PR or elsewhere.

Reply:
I mean issue not Github issue...

Reply:
I think restore the code won't solve the problem here. Say I have two paths `/symbolic_dir/non_exist_data` and `/nvme0n1/non_exist_data`, the equivalence check should still fails.